src/transformers/models/auto/configuration_auto.py

```diff
@@ -176,6 +176,7 @@
         ("pop2piano", "Pop2PianoConfig"),
         ("prophetnet", "ProphetNetConfig"),
         ("pvt", "PvtConfig"),
+        ("pvt_v2", "PvtV2Config"),
```

Comment:
The naming format is mixed in this file, for instance the mobilenet_v1 and mobilenet_v2 use snake case. Another popular implementation of the PVTv2 model is in the timm library where is is referred to as pvt_v2, and also that's how it appears in the file tree, so I thought snake case seemed more appropriate. Is there a hard expectation to hyphenate this?

Reply:
Revisiting this, since the most commonly used implementation of pvt_v2 is currently in the timm library under that name, I think people would appreciate consistency between these libraries. What do you think?

Reply:
OK - let's go for the underscore for consistency with timm here